leetcode/0169.Majority-Element/169. Majority Element.go

```diff
@@ -6,12 +6,11 @@ func majorityElement(nums []int) int {
 	for i := 0; i < len(nums); i++ {
 		if count == 0 {
 			res, count = nums[i], 1
+		}
+		if nums[i] == res {
+			count++
 		} else {
-			if nums[i] == res {
```

Comment:
This fix is not correct. if count != 0, then it will judge else case. You ignore else case, directly write if-else. I have corrected these code.